utils/swift_build_support/swift_build_support/build_script_invocation.py

```diff
@@ -669,7 +673,13 @@ def execute(self):
                 #
                 # This just maintains current behavior.
                 if index > last_impl_index:
-                    self._execute(pipeline, [self.args.host_target])
+                    non_darwin_cross_compile_hostnames = [
```

Comment:
@buttaface can you add a comment here (or fix up the comment above) explaining that we support it for non-darwin platforms. I am fine with a follow on commit fixing the comment.

Reply:
Will do.

Reply:
I simply removed the comments above, since it is no longer true. Darwin platforms were [already cross-compiling in a different way](https://github.com/apple/swift/pull/36917/files#diff-57b5bec593d589a8abf555dcffcdd0748b1dc27cbc7beec0b0891c41fd274909R133), and this change adds support for non-Darwin platforms.